Kerdy/Kerdy/MyProfile/VC/ActivityEditHalfVC.swift

```diff
@@ -0,0 +1,292 @@
+    private func setUI() {
+        view.backgroundColor = .clear
+    }
+    
+    private func setTabGesture() {
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(self.handleCloseAction))
+        dimmedView.addGestureRecognizer(tapGesture)
+    }
+    
+    private func setLayout() {
+        view.addSubview(dimmedView)
+        view.addSubview(containerView)
+        
+        dimmedView.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+        containerView.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview()
+        }
+        containerView.addSubview(baseView)
+    
+        baseView.addSubviews(clubBtn, educationBtn, titleLabel, notesLabel, grabBar, nextBtn)
+        
+        grabBar.snp.makeConstraints {
+            $0.width.equalTo(44)
+            $0.height.equalTo(4)
+            $0.top.equalToSuperview().offset(8)
+            $0.centerX.equalToSuperview()
+        }
+        
+        titleLabel.snp.makeConstraints {
+            $0.top.equalToSuperview().offset(50)
+            $0.leading.equalToSuperview().offset(17)
+        }
+        
+        notesLabel.snp.makeConstraints {
+            $0.top.equalTo(titleLabel.snp.bottom).offset(10)
+            $0.leading.equalToSuperview().offset(17)
+        }
+        
+        clubBtn.snp.makeConstraints {
+            $0.height.equalTo(60)
+            $0.leading.equalToSuperview().offset(17)
+            $0.top.equalTo(notesLabel.snp.bottom).offset(24)
+            $0.trailing.equalTo(view.snp.centerX).offset(-3)
+        }
+        
+        educationBtn.snp.makeConstraints {
+            $0.height.equalTo(60)
+            $0.leading.equalTo(view.snp.centerX).offset(3)
+            $0.trailing.equalToSuperview().offset(-17)
+            $0.top.equalTo(notesLabel.snp.bottom).offset(24)
+        }
+        
+        nextBtn.snp.makeConstraints {
+            $0.height.equalTo(60)
+            $0.leading.equalToSuperview().offset(17)
+            $0.trailing.equalToSuperview().offset(-17)
+            $0.bottom.equalToSuperview().offset(-25)
+        }
+        
+        baseView.snp.makeConstraints { make in
+            make.top.equalTo(containerView)
+            make.bottom.equalTo(containerView)
+            make.leading.equalTo(containerView)
+            make.trailing.equalTo(containerView)
+        }
+        
+        containerViewHeightConstraint = containerView.heightAnchor.constraint(equalToConstant: defaultHeight)
+        containerViewBottomConstraint = containerView.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: defaultHeight)
+        containerViewHeightConstraint?.isActive = true
+        containerViewBottomConstraint?.isActive = true
+
+    }
+    
+    @objc func clubBtnTapped(_ sender: UIButton) {
+        if clubBtn.isSelected {
+            clubBtn.isSelected = false
+            clubBtn.layer.borderColor = UIColor(named: "kerdy_gray01")?.cgColor
```

Comment:
```suggestion
            clubBtn.layer.borderColor = UIColor.kerdyGray01.cgColor
```
과 같이 사용하면 좋을 것 같습니다!